### ISSUE TITLE
Luminosity ratio of keyboard focus indicator on 'Places to buy' control is less than 3:1

### ISSUE THREAD

**Issue by 232amansingh** [microsoft/BotFramework-WebChat#5742]
### Is it an issue related to Adaptive Cards?

Yes, this is related to Adaptive Card. However, it does not repro on Adaptive Cards Designer.

### What is the PWD impact?

When the keyboard focus indicator on the “Places to buy” control has a luminosity contrast ratio less than 3:1, users with low vision or those in high‑contrast settings cannot reliably perceive where focus is.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Repro Steps: ​           
1. Open URL: Web Chat in edge browser.
2. Ask query 'Carousel'.
3. Navigate to Places to buy control.
4. Observe the issue that Luminosity ratio of keyboard focus indicator on 'Places to buy' control is less than 3:1

### What do you expect?

Luminosity ratio of keyboard focus indicator on 'Places to buy' control should be equal or greater than 3:1

### What actually happened?

Luminosity ratio of keyboard focus indicator on 'Places to buy' control is less than 3:1

### Do you have any screenshots or recordings to repro the issue?

<img width="1920" height="1020" alt="Image" src="https://github.com/user-attachments/assets/91b02079-b7cc-4ed0-90f6-23ec25d21538" />

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

### Adaptive Card JSON

```json

```
